pyfr/backends/base/backend.py

```diff
@@ -55,7 +55,7 @@ def __init__(self, cfg):
     def lookup(self):
         pkg = f'pyfr.backends.{self.name}.kernels'
         dfltargs = dict(alignb=self.alignb, fpdtype=self.fpdtype,
-                        soasz=self.soasz, math=math)
+                        soasz=self.soasz, aosoasz=self.aosoasz, math=math)
```

Comment:
I'd pick a slightly different number to aosoasz, as it is more than that (it is an entire block that also impacts the rows).

Reply:
Actually I only use soasz*aosoasz multiplication, never aosoasz itself. So maybe I can pass this multiplication instead, and name it sz.